swagger_server/controllers/operand_controller.py

```diff
@@ -22,7 +36,11 @@ def delete_operand(operand_id):  # noqa: E501
 
     :rtype: None
     """
-    return 'do some magic!'
+    try:
+        operand = Operand.delete().where(Operand.id == operand_id)
```

Comment:
https://github.com/math-as-a-service/configurations/pull/19/files#diff-2226ea29575172638ec6566a5c3c38a7R34 <-- maybe a slightly different way to approach it